circleguard/draw.py

```diff
@@ -78,10 +86,26 @@ def init():
         def update(i):
             plot1.set_data(data1[0][i - 100:i], data1[1][i - 100:i])
             plot2.set_data(data2[0][i - 100:i], data2[1][i - 100:i])
+            if i != 0 and i % int((total/10)) == 0:
+                print(f"Saved {round(100 * float(i)/float(total))}% already!")
```

Comment:
replace with 
`print(f"Saved {round(100 * float(i)/float(total))}% of the video")`

Reply:
done